dm/dm/master/server.go

```diff
@@ -575,7 +575,7 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 
 		resp.Result = true
 		if cfg.RemoveMeta {
-			resp.Msg += "`remove-meta` in task config is deprecated, please use `start-task ... --remove-meta` instead"
+			resp.Msg = "`remove-meta` in task config is deprecated, please use `start-task ... --remove-meta` instead"
```

Comment:
please add a comment that what message will be overwritten

Reply:
no message will be overwritten. The resp.Msg is just assigned once. Because this [pr](https://github.com/pingcap/tiflow/pull/4568/files)

```go
	msg, err := checker.CheckSyncConfigFunc(ctx, stCfgs, ctlcommon.DefaultErrorCnt, ctlcommon.DefaultWarnCnt)
	if err != nil {
		resp.CheckResult = terror.WithClass(err, terror.ClassDMMaster).Error()
		return resp, nil
	}
	resp.CheckResult = msg
```

`resp.Msg` is changed to `resp.CheckResult`.

So I just remove `+` from here.